package.json

```diff
@@ -70,19 +70,19 @@
     "randomstring": "^1.1.5",
     "raven": "^2.6.4",
     "redis": "^3.0.2",
-    "request": "^2.85.0",
+    "request": "^2.88.2",
     "resin-device-config": "^5.1.1",
     "resin-semver": "^1.6.0",
-    "rsmq": "^0.11.0",
+    "rsmq": "^0.12.0",
```

Comment:
https://github.com/smrchy/rsmq/blob/v0.12.0/CHANGELOG.md#0120

Basically just drops support for old versions of nodejs by bump to redis@3 (which we've already bumped to and notes performance benefits in its changelog)